File: IQ/Classes/clsResourceCategory.vb
Option Explicit On
Option Strict On

Imports dataAccess
Imports System.IO

' Represents a Resource category for display on the Resources page
' TODO: needs to implement i_Editable

Public Class clsResourceCategory

    Public ID As Integer
    Public Name As String
    Public Translation As clsTranslation
    Public Order As Integer
    Public Resources As List(Of clsResource)

    Public Sub New()

    End Sub

    Public Sub New(ByVal ID As Integer, ByVal Name As String, ByVal Translation As clsTranslation, ByVal Order As Integer)

        Me.ID = ID
        Me.Name = Name
        Me.Translation = Translation
        Me.Order = Order

    End Sub

End Class

File: IQ/Classes/clsResourceCategory.cs
using dataAccess;
using System.IO;



// Represents a Resource category for display on the Resources page
// TODO: needs to implement i_Editable

public class clsResourceCategory
{

    public int ID;
    public string Name;
    public clsTranslation Translation;
    public int Order;
    public List<clsResource> Resources;

    public clsResourceCategory()
    {

    }

    public clsResourceCategory(int ID, string Name, clsTranslation Translation, int Order)
    {

        this.ID = ID;
        this.Name = Name;
        this.Translation = Translation;
        this.Order = Order;

    }

}